code/src/components/ThoughtsElement.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import moment from 'moment';
+
+const ThoughtsElement = ({ message, onHeartLikes}) => {
+  return (
+    <div
+      className='chat-section'
+      key={message._id}
+      >
+      <h4>{message.message}</h4>
+      <div className='likes-section'>
+        <div className='likes'>
+        <button className={message.hearts === 0 ? 'button-grey' : 'button-pink'}
+          onClick={() => onHeartLikes(message._id)}>
+          <span className='heart' role="img" aria-label="heart-icon">❤️</span>
+        </button>
+        </div>
+        <p className='counter' tabIndex="0">
+          x {message.hearts}
+        </p>
+        <p className='time' tabIndex="0">{moment(message.createdAt).fromNow()}</p>
```

Comment:
I'm not super sure about this, but shouldn't tabindex only be used for elements that the user can interact with? Like when you're creating a custom button or input using a div with an onclick, because those don't have the inbuilt tabindex like the button tag? Maybe the code coach can help us here ;)